### PR TITLE
Remove preventInput flag on requestAnimationFrame

### LINKED ISSUE #328: Prevent invalid input broken on IE11
After https://github.com/vaadin/vaadin-text-field/pull/326 you can actually type an invalid character into the text field (once after focus) even if the field is marked `preventInvalidInput`.

### PR DESCRIPTION
**Pull request by yuriy-fix**
Fixes #328

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-text-field/329)

### CODE REVIEW

**Comment by tomivirkki** [comment]
No test?

**Comment by yuriy-fix** [comment]
Have come up with an idea of the test with the scenario described in the ticket.

File: test/text-field.html [hunk 1]
```diff
@@ -347,6 +347,22 @@
           });
         });
 
+        (IE ? it : it.skip)('should prevent invalid if input event is user originated', done => {
+          textField.preventInvalidInput = true;
+          textField.pattern = '[0-9]*';
+
+          textField.dispatchEvent(new Event('focusin'));
+
+          setTimeout(() => {
```
Comment (tomivirkki) [inline test/text-field.html]: The fix uses raf but the test uses timeout. That sounds like a recipe for a flakey test.

Comment (yuriy-fix) [reply test/text-field.html]: Good catch! Replaced with `requestAnimationFrame`.

**Comment by yuriy-fix** [comment]
Removing flag in the `onInput` as well.

**Pull request merged**